### PR TITLE
Fix publish script

### PR DESCRIPTION
**Pull request by nikhilvenkatraman-okta**
- Fix publish script for version 1.17.0

@manueltanzi-okta @robertdamphousse-okta @jgao-okta

### CODE REVIEW

File: scripts/publish.sh [hunk 1]
```diff
@@ -5,6 +5,8 @@ source $OKTA_HOME/$REPO/scripts/setup.sh
 REGISTRY="${ARTIFACTORY_URL}/api/npm/npm-okta"
 
 export TEST_SUITE_TYPE="build"
+yarn global add @okta/ci-update-package
```
Comment (haishengwu-okta) [inline scripts/publish.sh]: hmm, how does it work before when publishing 2.x?

Comment (nikhilvenkatraman-okta) [reply scripts/publish.sh]: I am not sure how it worked before. With this change I am trying to solve an issue we are having the publish script not being able to run with version 1.17.0.  @manueltanzi-okta pointed me to the latest publish script that has these change https://github.com/okta/okta-auth-js/blob/master/scripts/publish.sh#L11

**Comment by manueltanzi-okta** [comment]
I think would be safer to just directly substitute with the newest publish script: https://github.com/okta/okta-auth-js/blob/master/scripts/publish.sh and probably also the setup script: https://github.com/okta/okta-auth-js/blob/master/scripts/setup.sh rather than just adding these two lines, since I saw there were other changes in the past. What @jgao-okta thinks?

**Comment by nikhilvenkatraman-okta** [comment]
I am ok to update the scripts to the latest version, since there is no easy way to tell if these will run correctly on CI.

**Comment by jgao-okta** [comment]
I don't have a strong preference, but I do think that if the current script in master works, it would be safer to substitute it.

File: scripts/publish.sh (hunk 1 above)
Comment (haishengwu-okta) [reply scripts/publish.sh]: I see. because you're fixing on the release branch.

**Pull request merged**